### ISSUE TITLE
Vim c key (Change) support

### ISSUE THREAD

**Issue by johanhelsing** [lapce/lapce#772]
Vim keybindings are already pretty good, but I keep hitting this one, it's pretty ingrained in my muscle memory.

For instance:

- `cc` delete current line and enter insert mode at correct indentation level
- `cw` delete rest of the word and enter insert mode
- `ciW` delete until whitespace in both direction and enter insert mode
- `ci"` delete inside quotes and enter insert mode
- `C` delete rest of the line and enter insert mode

Already mentioned in #700, but it's been said it's better to make separate issues to track missing vim features individually.

**Comment by anenankhu** [comment]
I think `ce` does the same thing as `cw`, but nevertheless should be not forgotten too.

**Comment by kaifastromai** [comment]
Any updates on this @jonbenronron? It is glaringly absent.

**Comment by jonbenronron** [comment]
@TudbuT  Yes go ahead. I'll unassign myself from this task. I've had unexpected urgent tasks at the university and at work so sadly I haven't had time to finish this in my free time.

**Comment by TudbuT** [comment]
Here ya go!

**Comment by CyanBlob** [comment]
@TudbuT Have you started working on the "change inside X" yet? I use ci" and ci{ religiously. Thanks for the work so far!

**Comment by TudbuT** [comment]
> @TudbuT Have you started working on the "change inside X" yet? I use ci" and ci{ religiously. Thanks for the work so far!

sadly not, but i will start doing that soon if you want

**Comment by eleijonmarck** [comment]
this is such a crucial feature for vim like behavior, use it all the time! hope you get it in soon so i can switch from vscode
